test/IISIntegration.FunctionalTests/HelloWorldTest.cs

```diff
@@ -20,31 +19,23 @@ public HelloWorldTests(ITestOutputHelper output) : base(output)
         {
         }
 
-        [ConditionalTheory]
-        [OSSkipCondition(OperatingSystems.Linux | OperatingSystems.MacOSX)]
-        [FrameworkSkipCondition(RuntimeFrameworks.CoreCLR)]
```

Comment:
Because the framework we're actually testing runs out of process, It doesn't matter as much which framework is used to run the test. These skips were added to keep the same test from executing twice, one for each runner framework.